### DISCUSSION TITLE
Universal binaries for macOS

### DISCUSSION THREAD

**Discussion by DidierMalenfant** [BartmanAbyss/vscode-amiga-debug#235] [Ideas]
Just want to make sure this hasn't been attempted or shot down before I give it a try... We could build the FS-UE, gcc, etc... binaries as Universal binaries for Apple Silicon Macs.

I've had issues compiling one of gcc's dependencies on M2 for my own project but I can revisit this again if that's cool with everyone.

**Comment by BartmanAbyss** [comment]
Sure, I appreciate the support. @grahambates did all of the Linux/MacOS ports

**Reply by DidierMalenfant** [reply]
I'm going to reach out to him and see if I can ask a few questions then 👍🏻

**Comment by DidierMalenfant** [comment]
I have the version of FS-UAE used in the extension compiled as Universal.

Now attacking the toolchain...
